src/Raven.Server/Web/System/DatabasesHandler.cs

```diff
@@ -83,6 +87,77 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        [RavenAction("/topology/modify", "POST", AuthorizationStatus.Operator)]
```

Comment:
@ppekrol I think we need to change the URL here... 
What do you think about `/admin/databases/topology/modify`?

Reply:
Done